core/meta/src/TClass.cxx

```diff
@@ -488,6 +488,49 @@ DeclIdMap_t *TClass::GetDeclIdMap() {
 #endif
 }
 
+
+namespace {
+
+////////////////////////////////////////////////////////////////////////////////
+/// Count the number of occurrences of needle in haystack.
+
+static int CountStringOccurrences(TString needle, TString haystack) {
```

Comment:
In principle this could lead to false positive/count as it does not respect delimiters (`,<> `).  eg. `MyTemplate<MyDouble32_t>` albeit I assume it would be rare.

Reply:
The alternative would be to use `TClassEdit`. Would you recommend that? Do we have the interfaces to do that? I suppose I'd need to recurse over template args to check whether they themselves are templates?

Reply:
There might be a cheap improvement by checking that the characters surrounding the needle are delimiters:
```
if (  (posFound == 0 || isdelimin(haystack[posFound-1]) && (posFound+strlen(needle) >= haystack.Length() || isdelimin(haystack[posFound+strlen(needle)])
{
   +numOccurrences;
}
```
with 
```
bool isdelimit(char c) {
   return (c == ',' || c == '[' || c == ']' || c == ' ');
}
```